com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -113,6 +119,23 @@ public static GUIStyle miniLabelButton
         /// <summary>RenderPipeline Global Settings icon</summary>
         public static readonly Texture2D globalSettingsIcon;
 
+        /// <summary>
+        /// Gets the icon that describes the <see cref="MessageType"/>
+        /// </summary>
+        /// <param name="messageType">The <see cref="MessageType"/> to obtain the icon from</param>
+        /// <returns>a <see cref="Texture2D"/> with the icon for the <see cref="MessageType"/></returns>
+        internal static Texture2D GetMessageTypeIcon(MessageType messageType)
+        {
+            return messageType switch
+            {
+                MessageType.None => m_TransparentTexture,
```

Comment:
Shouldn't this be null instead? (transparent texture seams a unneeded waste of memory)

Looking into code, 
> EditorGUIUtility.TrTextContentWithIcon(text, icon)

 is just sugar to write
> EditorGUIUtility.TrTextContent(text, icon: icon)

which fully support null icon as the default value is in fact `null`.